cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -59,22 +59,37 @@ def get_response(self, form):
 
     def success(self):
         if self.request.is_ajax():
-            return JsonResponse({'result': 'pass'})
+            if get_feedback_type(self.block_value) == 'suggestion':
+                return JsonResponse(
+                    {'result': 'pass',
+                     'heading': 'Thank you!',
+                     'message': "Be sure to also sign up for our email list "
```

Comment:
The radio-button suggestion format is indeed OAH-specific, and all other feedback forms should get the default success message. I think this could be part of a radio-button framework discussed earlier that is out of scope for this effort.